Header header
bool stopped
float32 angle
float32 curr_vel
float32 des_vel